rama-socks5/src/proto/client.rs

```diff
@@ -0,0 +1,617 @@
+//! Client implementation of the SOCKS5 Protocol [RFC 1928]
```

Comment:
you were on the socks5 feature branch it seems. Remove these files again please.